src/Http/Controllers/GoogleAnalyticsController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tightenco\NovaGoogleAnalytics\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Arr;
+use Tightenco\NovaGoogleAnalytics\File;
+use Illuminate\Routing\Controller;
+use Spatie\Analytics\Analytics;
+use Spatie\Analytics\Period;
+
+class GoogleAnalyticsController extends Controller
+{
+    public function index(Request $request)
+    {
+        $limit = $request->input('limit', 10);
+        $page = $request->input('page', 1);
+        $offset = ($page - 1) * $limit;
+        $duration = $request->input('duration', 'week');
+        $searchTerm = $request->input('s', null);
+        $sortBy = $request->input('sortBy', 'ga:pageviews');
+        $sortDirection = $request->input('sortDirection', 'desc');
```

Comment:
Can we instead use the same sort syntax google uses? `sortCol` for asc, `-sortCol` for desc? (I haven't read the Vue code yet so I'll probably know the answer shortly)

Reply:
(looking at the Vue, I'm guessing this particularly style is how Nova does it; if so, that's fine!)